src/ClientResolver.php

```diff
@@ -425,7 +425,7 @@ public static function _apply_validate($value, array &$args, EmitterInterface $e
 
     public static function _default_api_provider()
     {
-        return new FilesystemApiProvider(__DIR__ . '/data');
+        return ApiProvider::defaultProvider();
```

Comment:
I think this can be removed in favor of providing the string "Aws\Api\ApiProvider::defaultProvider" to the "fn" attribute of this option.